### PR TITLE
Doctor Visit and Online Consultation Popup Dialog

### PR DESCRIPTION
**Pull request by Chinnu1103**

### CODE REVIEW

**Comment by Chinnu1103** [comment]
@Shre26ya It gives me an error when I try to book an online consultation saying that book_appointment is not correct link

**Comment by Chinnu1103** [comment]
@sash7410 can you change the Doctor Visit Appointment to Online Consultation appointment?

File: MediLink/doctor/migrations/__pycache__/0001_initial.cpython-311.pyc
Comment (sash7410) [inline MediLink/doctor/migrations/__pycache__/0001_initial.cpython-311.pyc:1]: remove this file

File: MediLink/doctor/migrations/__pycache__/__init__.cpython-311.pyc
Comment (sash7410) [inline MediLink/doctor/migrations/__pycache__/__init__.cpython-311.pyc:1]: do we need this file

File: MediLink/hospital/migrations/__pycache__/0001_initial.cpython-311.pyc
Comment (sash7410) [inline MediLink/hospital/migrations/__pycache__/0001_initial.cpython-311.pyc:1]: do we need this file

File: MediLink/hospital/migrations/__pycache__/__init__.cpython-311.pyc
Comment (sash7410) [inline MediLink/hospital/migrations/__pycache__/__init__.cpython-311.pyc:1]: do we need this file

File: MediLink/user/migrations/__pycache__/0001_initial.cpython-311.pyc
Comment (sash7410) [inline MediLink/user/migrations/__pycache__/0001_initial.cpython-311.pyc:1]: do we need this file

File: MediLink/user/migrations/__pycache__/__init__.cpython-311.pyc
Comment (sash7410) [inline MediLink/user/migrations/__pycache__/__init__.cpython-311.pyc:1]: do we need this file

**Review by sash7410 — APPROVED** [review]

**Pull request merged**